settings.py

```diff
@@ -32,12 +33,15 @@
 # "gpt4": 8192 tokens
 # "GoogleFlan": ? tokens
 # "llama2": ? tokens
-# LLM_MODEL_TYPE must be one of "llama2", "GoogleFlan"
 LLM_MODEL_TYPE = "gpt35"
-# EMBEDDINGS_PROVIDER must be one of: "openai", "huggingface"
+# API_URL must be the URL to your (local) API
+# If LLM_TYPE is "local_llm" and model is run on your local machine, API_URL should be "localhost:11434" by default
+# API_URL = "localhost:11434"
+# EMBEDDINGS_PROVIDER must be one of: "openai", "huggingface", "local_embeddings"
 EMBEDDINGS_PROVIDER = "openai"
 # EMBEDDINGS_MODEL must be one of: "text-embedding-ada-002", "all-mpnet-base-v2"
 # If EMBEDDINGS_MODEL is "all-mpnet-base-v2" then EMBEDDINGS_PROVIDER must be "huggingface"
+# If EMBEDDINGS_MODEL is "local_embeddings" then EMBEDDINGS_PROVIDER must be one of the local models, e.g. "llama2"
```

Comment:
Hi @Sennaa, @lrvl, I assume this line should be "If EMBEDDINGS_PROVIDER is "local_embeddings" then EMBEDDINGS_MODEL must be one of the local models e.g. "llama2" ?
Is "llama2" the LLM ánd embedding model you actually used for testing the user story?

Reply:
Hi @StefanTroost . Yes, that's correct my current local settings.py uses EMBEDDINGS_MODEL = "llama2". During the user story we first tested with "orca-mini" and after that for a higher quality with "llama2".

I'd like to suggest a small change to how we handle settings.py in our GitHub repo in order to enhance security and personalization:

1. Rename `settings.py` to `settings_template.py` : This becomes a base template.
2. Add `settings.py` to `.gitignore`: Each developer creates their own settings.py from the template, which avoids sensitive data getting into the repo.

This practice should make our configuration management safer and flexible. Thoughts?

Reply:
Hi @lrvl, that's a good idea indeed. I will first try to get the local LLM working together with the RAG pipeline on my Azure virtual machine (didn't succeed yesterday), and then do the changes regarding settings.py